src/components/site/Images.md

```diff
@@ -0,0 +1,22 @@
+```jsx
+<Images
+  imageSrc="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
+  content="Lorem Ipsum is simply dummy text of the printing and typesetting of Lorem Ipsum.industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five"
+></Images>
+```
+
+Html Snippet:
+```html
+<div>
+<div class="dg_image">
+  <img src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg" aria-hidden="true"></div>
+  <div class="dg_image left">
+    <img src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg" aria-hidden="true">
+  </div>
+  <p>Lorem Ipsum is simply dummy text of the printing and typesetting of Lorem Ipsum.industry. Lorem Ipsum has been the industry's standard dummy text ever   since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five</p>
+  <div class="dg_image right">
```

Comment:
Please move this class to the image, and remove the div. add alt tag and remove aria-hidden